app/Models/Customer.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Models;
+
+use App\DB\Model;
+use App\Core\DB;
+
+class Customer extends Model
+{
+    private ?int $id;
+    private ?string $name;
+    private ?string $email;
+
+    public function __construct( ?int $id = null )
+    {
+        $this->id = $id;
+        if ($id != null) {
+            $this->getCustomerFromDatabase();
+        }
+    }
+
+    public function getId():?int
+    {
+        return $this->id;
+    }
+
+    public function getName():?string
+    {
+        return $this->name;
+    }
+
+    public function setName(?string $name):void
+    {
+        $this->name = $name;
+    }
+
+    public function getEmail():?string
+    {
+        return $this->email;
+    }
+
+    public function setEmail(?string $email):void
+    {
+        $this->email = $email;
+    }
+
+    private function save():void

```

Comment:
bu method private olmali deyil, public olmalidi. Private methodlari diger classlar cagira bilmir: https://www.w3schools.com/php/php_oop_access_modifiers.asp#:~:text=public%20%2D%20the%20property%20or%20method,be%20accessed%20within%20the%20class